pkg/arvo/sys/vane/gall.hoon

```diff
@@ -783,9 +837,24 @@
       [p q]:beak
     ::
     ?:  ?=(%raw-poke -.deal)
-      =/  =schematic:ford  [%vale ship-desk +.deal]
-      =/  =note-arvo  [%f %build live=%.n schematic]
-      (mo-pass path note-arvo)
+      ::  TODO: Remove this horrific hack when ford pinto comes!
+      ?+  mark.deal
+        =/  =schematic:ford  [%vale ship-desk +.deal]
+        =/  =note-arvo  [%f %build live=%.n schematic]
+        (mo-pass path note-arvo)
+      ::
+          %chat-action
+        =/  chat-act=(unit chat-action)
+          ((soft chat-action) ;;(chat-action noun.deal))
+        ?~  chat-act
+          ~&  %gall-raw-poke-chat-action-failed
```

Comment:
I would add `agent` and `attributing.routes` to this printf so we know which ship is sending to which app.